### DISCUSSION TITLE
Character Restore

### DISCUSSION THREAD

**Discussion by ghost** [chromiecraft/chromiecraft#5455] [General]
Hello there, 

could my character Noctoru be restored pretty please?

Kind regards

**Comment by 55Honey** [comment]
Hello @Miha1515 ,

You're looking for:
https://www.chromiecraft.com/product/restore-deleted-character/

The necessary ChromiePoints can be obtained in various ways. Our [contributors](https://www.chromiecraft.com/en/how-to-contribute/) earn them for example by testing, reporting bugs, or donating. 

Regards
